.github/workflows/jira.yaml

```
@@ -59,7 +59,7 @@ jobs:
           echo "type=ISS" >> $GITHUB_OUTPUT
         fi
     - name: Create ticket
-      if: !(github.event.pull_request && contains(github.event.pull_request.labels.*.name, 'dependencies')) && (github.event.action == 'opened' || github.event.action == 'created')
+      if: (!(github.event.pull_request && contains(github.event.pull_request.labels.*.name, 'dependencies'))) && (github.event.action == 'opened' || github.event.action == 'created')
```

I'm not sure where I got 'created' from initially, but looks like we can simplify by only looking for 'opened' as the action. It's consistent across issues and pull requests:
- https://docs.github.com/en/webhooks-and-events/webhooks/webhook-events-and-payloads?actionType=opened#issues
- https://docs.github.com/en/webhooks-and-events/webhooks/webhook-events-and-payloads?actionType=opened#pull_request

Untested suggestion:

```suggestion
      if: github.event.action == 'opened' && !(github.event.pull_request && contains(github.event.pull_request.labels.*.name, 'dependencies'))
```

---

Looks like 'created' maybe came from the `issue_comment` trigger up at the top in example usage. But it's not relevant to creating issues - that event should only trigger commenting on existing tickets. So I'm confident we can simplify.